provider/maas/environ_test.go

```diff
@@ -147,25 +147,12 @@ func (*environSuite) TestNewCloudinitConfig(c *gc.C) {
 	env, err := maas.NewEnviron(getSimpleCloudSpec(), cfg)
 	c.Assert(err, jc.ErrorIsNil)
 	script := expectedCloudinitConfig
-	cloudcfg, err := maas.NewCloudinitConfig(env, "testing.invalid", "quantal", []string{"eth0", "eth1"})
+	cloudcfg, err := maas.NewCloudinitConfig(env, "testing.invalid", "quantal")
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(cloudcfg.SystemUpdate(), jc.IsTrue)
 	c.Assert(cloudcfg.RunCmds(), jc.DeepEquals, script)
 }
 
-func (*environSuite) TestNewCloudinitConfigWithDisabledNetworkManagement(c *gc.C) {
```

Comment:
cause it is redundant with the previous test. There was one test for "with network changes" and one for "without network changes", but we don't have to do both if they are both "without network changes"